spec/models/post_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before :each do
+    @user = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher')
+    @post = Post.new(title: 'Hi', text: 'My world', comments_counter: 0, likes_counter: 0)
+  end
+
+  it 'checks title presence' do
+    @post.title = nil
+    expect(@post).to_not be_valid
+  end
```

Comment:
- [ ] For this test to pass, you will to update your post.rb model with validation "`Title must not be blank`"  please make sure the validation is added.